src/improvements/tasks/sep/016-U16-opcm-upgrade-v400-base/README.md

```diff
@@ -21,16 +21,77 @@ which reads the inputs from the [`config.toml`](./config.toml) file.
 
 ## Signing and execution
 
-Follow the instructions in the [Nested Execution](../../../NESTED.md) guide for the following steps:
+Signing and execution instructions for this task are unique because of the extra layer of nesting.
+The L1 ProxyAdmin Owner (L1PAO) is setup as following for Base Sepolia:
+
+```text
+┌───────┐┌───────┐┌───────┐┌───────┐┌───────┐┌───────┐
+│Signer1││Signer2││Signer3││Signer4││Signer5││Signer6│
+└┬──────┘└┬──────┘└┬──────┘└┬──────┘└┬──────┘└┬──────┘
+┌▽────────▽┐┌──────▽────────▽┐┌──────▽────────▽┐
+│Base      ││SC              ││FND             │
+└┬─────────┘└┬───────────────┘└┬───────────────┘
+┌▽───────────▽┐                │
+BaseNested    │                │
+└┬────────────┘                │
+┌▽─────────────────────────────▽┐
+│L1PAO                          │
+└┬──────────────────────────────┘
+┌▽─────────┐
+│ProxyAdmin│
+└──────────┘
+```
+
+where:
+
+- L1PAO is `0x0fe884546476dDd290eC46318785046ef68a0BA9`
+- FND is `0x6AF0674791925f767060Dd52f7fB20984E8639d8`
+- BaseNested is `0x646132A1667ca7aD00d36616AFBA1A28116C770A`
+- Base is `0x6AF0674791925f767060Dd52f7fB20984E8639d8` (on Sepolia, the Base and FND safes are the same)
+- SC is `0x5dfEB066334B67355A15dc9b67317fD2a2e1f77f`
+- `SignerN` represent arbitrary signers for Safes
+
+### Facilitator Instructions
```

Comment:
Is this the Facilitator or Task Creator's job?

Reply:
In 7d9995a3 I added a sentence that says "The facilitator, or someone acting on behalf of the facilitator, must perform the following steps:". I can imagine the task developer (e.g. us) creating the core task, and someone else (e.g. Base's facilitator or a dev) creating their approve hash task, or us doing it for them